<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>293</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Folder zawierający bazę danych pacjentów (SQLite):</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="pathDatabaseLEdit">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectPathDatabaseButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Wybierz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetPathDatabaseButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Folder zawierający badania OCT:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="pathOctExamLEdit">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectPathOctExamButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Wybierz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetPathOctExamButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Folder zawierający ręczną segmentację:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="pathManualSegmLEdit">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectPathManualSegmButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Wybierz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetPathManualSegmButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Folder zawierający automatyczną segmentację:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="pathAutoSegmLEdit">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectPathAutoSegmButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Wybierz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetPathAutoSegmButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="minimumSize">
        <size>
         <width>211</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>211</width>
         <height>91</height>
        </size>
       </property>
       <property name="title">
        <string>Wyświetlanie</string>
       </property>
       <widget class="QCheckBox" name="showCenterOnBscanCBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pokaż środek skanu na B-skanie</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showETDRSGridCBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pokaż siatkę ETDRS</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showBscanOnErrorPlotCBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pokaż B-skan na wykresie błędu</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>91</height>
        </size>
       </property>
       <property name="title">
        <string>Otwieraj badania na skanie:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="openBskanFirstRButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>pierwszym</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="openBskanMiddleRButton">
          <property name="text">
           <string>środkowym</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="dataSaveStructureLabel">
           <property name="text">
            <string>Struktura zapisu danych segmentacji:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dataSaveStructureCBox">
           <item>
            <property name="text">
             <string>xml</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>txt</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="blockPCVCBox">
         <property name="toolTip">
          <string>Blokuj rysowanie PCV poniżej warstwy ILM</string>
         </property>
         <property name="text">
          <string>Blokada PCV poniżej ILM</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="acceptButton">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
